examples/static_react_task/webapp/src/app.jsx

```diff
@@ -11,19 +11,76 @@ import ReactDOM from "react-dom";
 import { BaseFrontend, LoadingScreen } from "./components/core_components.jsx";
 import { useMephistoTask } from "mephisto-task";
 
+class ErrorBoundary extends React.Component {
```

Comment:
great first step in implementing this. the next step would be to move this component into `mephisto-task` and import it into this project from there. this will allow us to have a reusable mephisto component we can use.